includes/Core/Assets/Assets.php

```diff
@@ -835,20 +821,8 @@ private function get_inline_data() {
 			 *
 			 * @param array $data Authentication Data.
 			 */
-			'setup'         => apply_filters( 'googlesitekit_setup_data', array() ),
-
-			/**
-			 * Filters the notification message to print to plugin dashboard.
-			 *
-			 * @since 1.0.0
-			 *
-			 * @param array $data Notification Data.
-			 */
-			'notifications' => apply_filters( 'googlesitekit_notification_data', array() ),
-			'permaLink'     => $current_entity ? esc_url_raw( $current_entity->get_url() ) : false,
-			'pageTitle'     => $current_entity ? $current_entity->get_title() : '',
-			'publicPath'    => $this->context->url( 'dist/assets/js/' ),
-			'editmodule'    => $input->filter( INPUT_GET, 'editmodule', FILTER_SANITIZE_STRING ),
+			'setup'       => apply_filters( 'googlesitekit_setup_data', array() ),
+			'editmodule'  => $input->filter( INPUT_GET, 'editmodule', FILTER_SANITIZE_STRING ),
```

Comment:
`editmodule` still has a few references but we never actually use the query parameter anywhere (and I don't remember ever even seeing it used) so I would say we should remove this one too.

https://github.com/google/site-kit-wp/blob/29f4144035e710cbe8e9273f4509bc7b440ab189/assets/js/components/settings/SettingsModules.js#L42-L48

Reply:
Removed.